qt/settingsdialog.py

```diff
@@ -1018,6 +1022,25 @@ def __init__(self, parent):
         )
         layout.addWidget(self.cbCopyLinks)
 
+        # one file system option
+        self.cbOneFileSystem = QCheckBox(
+            _('Restrict to one file system'), self)
+        self.cbOneFileSystem.setToolTip(
+            'uses \'rsync --one-file-system\'\n'
+            'From \'man rsync\':\n'
+            + '\n'.join(textwrap.wrap(
+                'This tells rsync to avoid crossing a filesystem boundary '
+                'when recursing. This does not limit the user\'s ability '
+                'to specify items to copy from multiple filesystems, just '
+                'rsync\'s recursion through the hierarchy of each directory '
+                'that the user specified, and also the analogous recursion '
+                'on the receiving side during deletion. Also keep in mind '
+                'that rsync treats a "bind" mount to the same device as '
+                'being on the same filesystem.',
```

Comment:
longest tooltip I ever saw ;-)

Reply:
There are much more of this from-rsync-manpage-copy-and-pasted-tooltips. That is why I introduced the wrapping, to prevent translators from dealing with line breaks them selfs.